metrics/web/test/dashboard/presentation/widgets/build_result_bar_graph_test.dart

```diff
@@ -329,19 +330,19 @@ class _BuildResultBarGraphTestbed extends StatelessWidget {
 
   /// A list of [BuildResultViewModel] test data to test the [BuildResultBarGraph].
   static final buildResultBarTestData = [
```

Comment:
Should we test that the `BuildResultBarGraph` correctly parses the `BuildResultViewModel`s to the bar graph data?